javascript/stimulus_reflex.js

```diff
@@ -285,6 +292,8 @@ if (!document.stimulusReflexInitialized) {
     invokeLifecycleMethod('error', target, element)
     invokeLifecycleMethod('after', target, element)
   })
+  document.addEventListener('focus', receivedFocus, true)
```

Comment:
Only just learned about [focusin](https://developer.mozilla.org/en-US/docs/Web/API/Element/focusin_event) from your PR. I'm going to commit a patch here that switches over...

Reply:
Love all of the cool things I learn from @hopsoft :heart_eyes_cat: